Vineyard/Vineyard/GroupView.swift

```diff
@@ -121,6 +192,20 @@ struct GroupView: View {
                 }
             }
         }
+        .onAppear {
+            Task {
+                if group.people == nil || group.people!.isEmpty {
```

Comment:
maybe this question doesn't make sense, but should we not always fetch the groups people and resolutions on appear, regardless of whether it is nil or empty currently since a new person may have been added or there are new updates?